src/icons/AppIcon.tsx

```diff
@@ -0,0 +1,35 @@
+import { AppName } from "@/models/AppName";
+
+interface Props {
+  appName: AppName;
+}
+
+export default function AppIcon({ appName }: Props) {
```

Comment:
Take a look at `model/IconProps` and add the props to this one here. That way you can customize it by passing in values to `className`.